src/ecalc/libraries/libecalc/common/libecalc/dto/result/results.py

```diff
@@ -107,6 +108,8 @@ class CompressorResult(EquipmentResultBase):
     recirculation_loss: TimeSeriesRate
     rate_exceeds_maximum: TimeSeriesBoolean
     outlet_pressure_before_choking: TimeSeriesFloat
+    requested_inlet_pressure: Dict[datetime, TimeSeriesFloat]
+    requested_outlet_pressure: Dict[datetime, TimeSeriesFloat]
```

Comment:
should we have a snapshot test for DTO results as well? @jsolaas that would make it easier to also compare cli and dto results, that it has the same effect...

Reply:
Don't we have that in libecalc.common.tests.integration?

Reply:
I was not sure. Just saw that no snapshots were updated wrt these changes as far as I could see ...